statsig_rum/manifest.json

```diff
@@ -0,0 +1,49 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "86a69c7d-8042-4f93-96b1-f139b2058644",
+  "app_id": "statsig-rum",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Statsig RUM integration",
+    "title": "statsig-rum",
+    "media": [{
+      "media_type": "image",
+      "caption": "Statsig Logo",
+      "image_url": "images/statsig.png"
```

Comment:
You can remove this - the logo will automatically be added to the tile.

Reply:
done plz close